Gemfile

```diff
@@ -9,6 +9,7 @@ gem "haml", "~> 5.0", ">= 5.0.1"
 gem "nokogiri", "~> 1.8"
```

Comment:
Since there was a security issue with v1.8.0, lets use the recommended version constraint from rubygems.org which ensures that we must have at least v1.8.1 installed:

```ruby
gem 'nokogiri', '~> 1.8', '>= 1.8.1'
```
(but use double quotes 😉 )